compiler-utils/src/main/java/com/squareup/anvil/compiler/internal/PsiUtils.kt

```diff
@@ -172,7 +172,7 @@ public inline fun <reified T> KtAnnotationEntry.findAnnotationArgument(
   // First check if the is any named parameter. Named parameters allow a different order of
   // arguments.
   annotationValues
-    .mapNotNull { valueArgument ->
+    .firstNotNullOfOrNull { valueArgument ->
```

Comment:
Didn't realize I'd see this crazy method called in production code so soon 😄

Reply:
This name makes my eyes cross a little.

Reply:
The language lead indirectly replied to this here: https://twitter.com/relizarov/status/1390691109093920768?s=20

Reply:
True, naming is hard.